x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -58,6 +59,8 @@ const HomePageComponent: React.FC<HomePageProps> = ({ children }) => {
   const [showTimeline] = useShowTimeline();
 
   const { browserFields, indexPattern, indicesExist } = useSourcererScope();
+  // side effect: this will attempt to upgrade the endpoint package if it is not up to date
```

Comment:
❔ Elaborating to explain when/how often that will happen and why may be helpful to the reader.